generators/entity-client/templates/react/src/main/webapp/app/entities/blog/blog-update.tsx.ejs

```diff
@@ -0,0 +1,206 @@
+<%#
```

Comment:
this should be name using "entity" instead of blog in the same way entity templates are named for Angular. Check main generator but think these should begin with an underscore as well.